processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -248,7 +255,7 @@ public static <T> int getUncoveredIntervalsLimit(Query<T> query)
 
   public static <T> int getUncoveredIntervalsLimit(Query<T> query, int defaultValue)
   {
-    return parseInt(query, "uncoveredIntervalsLimit", defaultValue);
+    return parseInt(query, UNCOVERED_INTERVALS_LIMIT_KEY, defaultValue);
```

Comment:
Is there any reason for `parseInt` and `getAsInt` to remain different? Now that we have `getAsInt`, shall it replace `parseInt`?

(Btw, is the behavior the same? I haven't checked)